### DISCUSSION TITLE
How can I set label or other informations outside nodes?

### DISCUSSION THREAD

**Discussion by maymao** [neo4j-contrib/neovis.js#352] [Q&A] [answered]
I want to show informations like labels outside the node just like the example picture but I can only put it in the nodes? Is this function still available in 2.0 version?
![image](https://github.com/neo4j-contrib/neovis.js/assets/87053113/4945ea84-a8e1-44fe-b3a5-26777cf02834)

**Comment by thebestnom** [comment]
Yes it is, look at the migration guide, everything you can do in vis.js you can now do in neovis

**Reply by maymao** [reply]
Sorry where can I find the migration guide?

**Reply by maymao** [reply]
Is it the MIGRATING.md file?

**Reply by thebestnom** [reply]
Yep

**Reply by thebestnom** [reply]
Yep

**Reply by maymao** [reply]
Thanks so much!

I tried following the example codes but the labels are still inside my nodes. Could you take a look at my code, please?
```js
	labels: {
					User: {
						label: "userid",
						value: "pagerank",
						group: "communityId",
						[NeoVis.NEOVIS_ADVANCED_CONFIG]: {
							cypher: { 
								value: "MATCH (n) WHERE id(n) = $id RETURN n.weight AS c"
							},
							function: {
								title: NeoVis.objectToTitleHtml
							},
							static: {
								font: {
									size: 26,
									color: "#000FFF"
								},
							}
						}
					},
                    Community: {
                        label: "communityId",
                        [NeoVis.NEOVIS_ADVANCED_CONFIG]: {
							cypher: {
								value: "MATCH (n) WHERE id(n) = $id RETURN (10 - n.risk_level_count - n.risk_level_price) AS s"
							},
							function: {
								title: NeoVis.objectToTitleHtml
							}
                        }
                    }
				},
				relationships: {
					TRANSACTION: { 
                        value: "weight",
						[NeoVis.NEOVIS_ADVANCED_CONFIG]: {
							function: {
								title: NeoVis.objectToTitleHtml
							}
                        }
					},
                    BELONG_TO: {
						[NeoVis.NEOVIS_ADVANCED_CONFIG]: {
							function: {
								title: NeoVis.objectToTitleHtml
							}
                        }
					}
				},
```

**Reply by maymao** [reply]
sorry for the indention, it became messy after I paste

**Reply by thebestnom** [reply, marked as answer]
Ahh, if you want it outside the node Im pretty sure you need the shape to be `dot`
Also I recommend next time to put the code inside code block 😅
````
```js
Code here 
```
````
https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks

**Reply by maymao** [reply]
Thanks so much for your answer!! Sorry for the format, it was my first time use this discussion board, I truly appreciate your help!

**Reply by thebestnom** [reply]
No problem!
